platforms/openid-connect/yarn.lock

```diff
@@ -0,0 +1,3746 @@
+# This file is generated by running "yarn install" inside your project.
+# Manual changes might be lost - proceed with caution!
+
+__metadata:
```

Comment:
I think the auto build will fail with this version, could you regenerate the file with the v1 version same as azure platform?